docs/helloserver/loadgen/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/google-samples/istio-samples/helloserver/loadgen-base as final
```

Comment:
The reason we introduced this base image (`Dockerfile-base`): https://github.com/GoogleCloudPlatform/istio-samples/pull/58/files.
Basically, we wanted to speed up building of the `Dockerfile` for users of the tutorial.